src/components/SiteEventsModal.vue

```diff
@@ -42,20 +72,72 @@ export default {
                     sortable: true,
                 },
                 {
-                    field: 'date',
-                    label: 'date',
+                    field: 'UTC',
+                    label: 'UTC',
                     searchable: false,
                     sortable: true,
                 },
+                {
+                    field: 'user',
+                    label: 'user',
+                    visible: this.time_display_format == 'user',
+                    sortable: true,
+                },
+                {
+                    field: 'observatory',
+                    label: 'observatory',
+                    visible: this.time_display_format == 'observatory',
+                    sortable: true,
+                },
                 {
                     field: 'unix',
                     label: 'unix',
                     visible: false,
                     sortable: true,
                 },
-            ],
+            ]
+        },
+    },
+    data() {
+        return {
+            //site_events: [{} * 26],
+            site_events: JSON.parse(window.localStorage.getItem(`${this.sitecode}Events`)) || [],
+
+            time_display_format: "user", // 'user' or 'observatory',
         }
-    }
+    },
+    methods: {
+        async getSiteEvents() {
+            let url = `https://api.photonranch.org/api/events?site=${this.sitecode}`
+            let site_events = await axios.get(url)
+            site_events = site_events.data
+
+            // Function to convert from dublin julian days to unix time
+            let djd2unix = t => (t-25567.5)*86400*1000
+            let jd2unix = t => (t - 2440587.5) * 86400 * 1000
```

Comment:
Some magic numbers here. Could you make these into named variables? Or a descriptive comment would suffice.